src/transports/http.rs

```diff
@@ -229,10 +229,13 @@ impl<T> Response<T> {
     }
 }
 
+// We can do this because `hyper::client::ResponseFuture: Unpin`.
+impl<T> Unpin for Response<T> {}
```

Comment:
What about `T`?

Reply:
If we trust the compiler then this must be okay because it does compile. I think this is because we never construct a `Pin<T>` so whether T is Unpin doesn't matter. Although I am not sure why the compiler doesn't automatically implement Unpin in this case.